book/en/src/SUMMARY.md

```diff
@@ -23,13 +21,13 @@
     - [Inspecting generated code](./by-example/tips_view_code.md)
     - [Running tasks from RAM](./by-example/tips_from_ram.md)
     <!-- - [`#[cfg(..)]` support](./by-example/tips.md) -->
+- [RTIC vs. the world](./rtic_vs.md)
 - [Awesome RTIC examples](./awesome_rtic.md)
 - [Migration Guides](./migration.md)
   - [v0.5.x to v1.0.x](./migration/migration_v5.md)
   - [v0.4.x to v0.5.x](./migration/migration_v4.md)
   - [RTFM to RTIC](./migration/migration_rtic.md)
 - [Under the hood](./internals.md)
-  - [Cortex-M architectures](./internals/targets.md)
```

Comment:
Git mishap

Reply:
Just commented out stuff that is not ready for  the pre-release.